### PR TITLE
Mm/init final display message

### PR DESCRIPTION
**Pull request by MarcMcIntosh**
Change the final display message for the init script
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

**Review by chengoconnell — APPROVED** [review]

File: packages/init/__tests__/display-final-message.test.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { describe, test, afterEach, expect, beforeAll } from "@jest/globals";
```
Comment (Heyrows) [inline packages/init/__tests__/display-final-message.test.ts:1]: OPT:
As this step is just an element of display I don't think I would have made a test for that.
It subject to changes, way more than the other steps so I don't think as a developer I want to have to maintain that haha.

File: packages/init/src/steps/display-final-message.ts [hunk 2]
```diff
@@ -2,12 +2,30 @@ import { CONSTS } from "@slicemachine/core";
...
+export function displayFinalMessage(
+  cwd: string,
+  wasStarter: boolean,
+  reponame: string,
+  base: string
+): void {
   const yarnLock = NodeUtils.Files.exists(NodeUtils.YarnLockPath(cwd));
   const command = `${yarnLock ? "yarn" : "npm"} run ${CONSTS.SCRIPT_NAME}`;
-
   console.log();
```
Comment (Heyrows) [inline packages/init/src/steps/display-final-message.ts:13]: long forgotten console log ? haha

**Review by Heyrows — COMMENTED** [review]
The PR seems fine, since it includes the PR about no doc flag, there are still stuff to change :D

File: packages/init/src/steps/display-final-message.ts (hunk 2 above)
Comment (MarcMcIntosh) [reply packages/init/src/steps/display-final-message.ts:13]: It's there on purpose right?
I though it was making a new line :/

File: packages/init/__tests__/display-final-message.test.ts (hunk 1 above)
Comment (MarcMcIntosh) [reply packages/init/__tests__/display-final-message.test.ts:1]: It does/requires a few things..... I can check to see what assertions can be done to reduce the maintenance overhead

File: packages/init/src/steps/display-final-message.ts (hunk 2 above)
Comment (Heyrows) [reply packages/init/src/steps/display-final-message.ts:13]: haha no idea, if you think it's useful leave it, doesn't hurt anyone 😂

File: packages/init/__tests__/display-final-message.test.ts [hunk 3]
```diff
@@ -23,9 +23,7 @@ describe("display final message", () => {
     stdout.start();
     displayFinalMessage(TMP_DIR, false, FAKE_REPO, FAKE_BASE);
     stdout.stop();
-    expect(stdout.output).toContain(
-      "■ Run npm run slicemachine to launch Slice Machine and create your first Custom Type"
-    );
+    expect(stdout.output).toContain("npm run slicemachine");
```
Comment (Heyrows) [inline packages/init/__tests__/display-final-message.test.ts:26]: nice

**Review by Heyrows — APPROVED** [review]

**Pull request merged**